### PR TITLE
removed auto generated ids and name attributes

### PR DESCRIPTION
**Pull request by francodacosta**

### CODE REVIEW

File: web/templates/partial/offices.html [hunk 1]
```diff
@@ -29,39 +29,39 @@ <h3>
        </tr>
      </thead>
      <tr >
-       <td ><span id="g62" class="text-widget ">Current</span></td>
-       <td ><textarea id="g63" name="g63" cols="80" rows="6" maxlength="300" aria-required="true">xhkjhdkjfghjkdfhdjfhjksd</textarea></td>
-       <td ><input id="g64" name="g64" type="text" value="" size="30" maxlength="30" aria-required="false" /></td>
-       <td ><input id="g65" name="g65" type="text" value="" size="16" maxlength="16" aria-required="false" aria-describedby="hint147-content" /></td>
+       <td ><span  class="text-widget ">Current</span></td>
+       <td ><textarea  name="g63" cols="80" rows="6" maxlength="300" aria-required="true">xhkjhdkjfghjkdfhdjfhjksd</textarea></td>
```
Comment (breneser) [inline web/templates/partial/offices.html:33]: These still have the names?

File: web/templates/web/workbasket.html [hunk 2]
```diff
@@ -36,48 +36,48 @@ <h4>
             </div>
         </div>
         <div class="row">
-            <div class="twelve columns" ><input id="g28" name="g28" type="text" value="" size="8" class="tightForm" aria-required="false" /></div>
+            <div class="twelve columns" ><input  name="g28" type="text" value="" size="8" class="tightForm" aria-required="false" /></div>
```
Comment (breneser) [inline web/templates/web/workbasket.html:39]: names here as well.

File: web/templates/web/workbasket.html [hunk 3]
```diff
@@ -14,15 +14,15 @@ <h4>
             </div>
         </div>
         <div class="row">
-            <div class="twelve columns" ><input id="g26" name="g26" type="text" value="" size="10" class="tightForm" aria-required="false" /></div>
+            <div class="twelve columns" ><input  name="g26" type="text" value="" size="10" class="tightForm" aria-required="false" /></div>
```
Comment (breneser) [inline web/templates/web/workbasket.html:17]: name on this one

File: web/templates/web/workbasket.html [hunk 4]
```diff
@@ -14,15 +14,15 @@ <h4>
...
         </div>
         <div class="row">
             <div class="twelve columns" ><label id="pg27" for="g27" class="prompt north">Subject
                 </label>
             </div>
         </div>
         <div class="row">
-            <div class="twelve columns" ><select id="g27" name="g27" class="tightForm" aria-required="false">
+            <div class="twelve columns" ><select  name="g27" class="tightForm" aria-required="false">
```
Comment (breneser) [inline web/templates/web/workbasket.html:25]: this one as well

File: web/templates/web/workbasket.html (hunk 2 above)
Comment (breneser) [inline web/templates/web/workbasket.html:39]: last one with name

File: web/templates/web/workbasket.html [hunk 5]
```diff
@@ -36,48 +36,48 @@ <h4>
...
             </div>
         </div>
         <div class="row">
-            <div class="twelve columns" ><select id="g32" name="g32" class="tightForm" aria-required="true">
+            <div class="twelve columns" ><select  name="g32" class="tightForm" aria-required="true">
                     <option value="g32/0">All</option>
                     <option value="g32/1" selected="selected">For Attention</option>
                 </select></div>
         </div>
     </div>
```
Comment (breneser) [inline web/templates/web/workbasket.html:76]: These also have the name still

File: web/templates/partial/offices.html (hunk 1 above)
Comment (francodacosta) [reply web/templates/partial/offices.html:33]: i tried not to remove names on input fields, as I'm not sure if they might be needed when submitting

**Pull request merged**